imi/src/main/java/org/motechproject/nms/imi/domain/FileAuditRecord.java

```diff
@@ -27,10 +30,12 @@
 
```

Comment:
Is there any validation for the checkSum field?  Should a length for the column be provided?